src/EnergyPlus/StandardRatings.cc

```diff
@@ -563,7 +563,19 @@ namespace StandardRatings {
                     // Available chiller capacity as a function of temperature
                     AvailChillerCap = RefCap * ReformEIRChillerCapFT;
 
-                    ReformEIRChillerEIRFPLR = CurveValue(state, EIRFPLRCurveIndex, CondenserOutletTemp, reducedPLR);
+                    switch (state.dataCurveManager->PerfCurve(EIRFPLRCurveIndex)->numDims) {
+                    case 1:
+                        ReformEIRChillerEIRFPLR = CurveValue(state, EIRFPLRCurveIndex, CondenserOutletTemp);
+                        break;
+                    case 2:
+                        ReformEIRChillerEIRFPLR = CurveValue(state, EIRFPLRCurveIndex, CondenserOutletTemp, reducedPLR);
+                        break;
+                    case 3:
+                        ReformEIRChillerEIRFPLR = CurveValue(state, EIRFPLRCurveIndex, CondenserOutletTemp, reducedPLR, 0.0);
+                        break;
+                    default:
```

Comment:
can put `default:` immediately after `case 3:` since they are the same case.

Reply:
:+1: